core/src/main/java/nl/nn/adapterframework/align/Json2Xml.java

```diff
@@ -176,7 +176,7 @@ public boolean isNil(XSElementDeclaration elementDeclaration, JsonValue node) {
 			return null;
 		}
 		try {
-			Map<String, String> result=new HashMap<String,String>();
+			Map<String, String> result=new LinkedHashMap<String,String>();
```

Comment:
Op zich maakt dit niet uit toch? Json heeft geen ordering, mocht dit wel zo zijn, is misschien een comment handig :)

Reply:
comment toegevoegd:
// it is not really necessary to preserve the order, but often the results look nicer, and it is easier for testing ...